src/types/BeachballOptions.ts

```diff
@@ -5,87 +5,91 @@ import { ChangelogOptions } from './ChangelogOptions';
 
 export type BeachballOptions = CliOptions & RepoOptions & PackageOptions;
 
-export interface CliOptions {
+export interface CliOptions
```

Comment:
I like the alphabetizing but it makes diffing challenging--were any properties removed from here?

Reply:
No removals, apart from alphebetizing, the only change was to add `scope` to RepoOptions